### ISSUE TITLE
NUnit2045 should not generate warnings for implicit IDisposable scopes

### ISSUE THREAD

**Issue by unlikelier** [nunit/nunit.analyzers#912]
Warning `NUnit2045` generates unnecessary warnings when an implicit scope is used.

```cs
// OK
Asser.Multiple( () => {
  Assert.That( true );
  Assert.That( false );
} );

// OK
using( Assert.EnterMultipleScope ){
  Assert.That( true );
  Assert.That( false );
}

// Generates NUnit2045 but when it should not.
using var scope = Assert.EnterMultipleScope();
Assert.That( true );
Assert.That( false );

```

**Comment by manfred-brands** [comment]
Duplicate of #910
